super_rust_boy_c_binding/headers/rustboy.h

```diff
@@ -0,0 +1,25 @@
+#include <stdarg.h>
```

Comment:
Should this file be included? Isn't it generated as part of the build process?

Reply:
Removed all unused includes